roles/backup/tasks/awx-cro.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Get AWX custom resource object
+  k8s_info:
+    version: v1beta1
+    kind: AWX
+    namespace: '{{ meta.namespace }}'
+    name: '{{ tower_name }}'
+  register: _awx_cro
+
+- name: Set AWX object
+  set_fact:
+    _awx: "{{ _awx_cro['resources'][0] }}"
+
+- name: Set apiVersion
+  set_fact:
+    awx_api_version: "{{ _awx['apiVersion'] }}"
+
+- name: Set user specified spec
+  set_fact:
+    awx_spec: "{{ _awx['spec'] }}"
+
+- name: Template AWX object definition
+  template:
+    src: awx_object.yml.j2
+    dest: "_secrets/awx_object.yml"
```

Comment:
What's going to happen when there are more than 1 reconciliation loops running at once?

Reply:
Do we need to create a temp directory instead of writing this out to a hardcoded path?

Reply:
good point, I can add a random hash at the end of the `_secrets` directory name, same for `_definitions` for the restore role.